### PR TITLE
Add template to project continue

### PR DESCRIPTION
**Pull request by sonnn-1091**

### CODE REVIEW

**Comment by sonnn-1091** [comment]
ready

**Comment by sonnn-1091** [comment]
[o] rails_best_practices: success
[o] eslint: success
[o] rspec: success
[o] reek: success
[o] scss-lint: success
[o] bundle-audit: success
[o] rubocop: success
[o] brakeman: success

File: spec/views/static_pages/about.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe "static_pages/about.html.erb", type: :view do
+  include RSpecHtmlMatchers
+
+  it "has tag" do
+    render
+    expect(rendered).to have_tag("h1", :text => "About Me")
```
Comment (huongnguyenmta) [inline spec/views/static_pages/about.html.erb_spec.rb]: `have_tag("h1", text: "About Me")`

File: spec/views/static_pages/home.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe "static_pages/home.html.erb", type: :view do
+  include RSpecHtmlMatchers
+
+  it "has tag" do
+    render
+    expect(rendered).to have_tag("h1", :text => "Glog")
```
Comment (huongnguyenmta) [inline spec/views/static_pages/home.html.erb_spec.rb]: `have_tag("h1", text: "Glog")`

**Comment by huongnguyenmta** [comment]
commented

**Comment by SonNguyen1604** [comment]
ready 
[o] scss-lint: success
[o] eslint: success
[o] bundle-audit: success
[o] reek: success
[o] brakeman: success
[o] rails_best_practices: success
[o] rubocop: success
[o] rspec: success

**Comment by sonnn-1091** [comment]
ready

**Comment by sonnn-1091** [comment]
[o] scss-lint: success
[o] eslint: success
[o] bundle-audit: success
[o] reek: success
[o] brakeman: success
[o] rails_best_practices: success
[o] rubocop: success
[o] rspec: success

**Pull request merged**